account_financial_report/models/account_move_line.py

```diff
@@ -32,9 +50,9 @@ def init(self):
             )
 
     @api.model
-    def search_count(self, args):
+    def search_count(self, domain, limit=None):
         # In Big DataBase every time you change the domain widget this method
         # takes a lot of time. This improves performance
         if self.env.context.get("skip_search_count"):
             return 0
-        return super(AccountMoveLine, self).search_count(args)
+        return super().search_count(domain, limit)
```

Comment:
Always respect keyword arguments:

```suggestion
        return super().search_count(domain, limit=limit)
```

Reply:
Ok